services/quizzes/src/components/SubmissionComponents/MultipleChoice.tsx

```diff
@@ -116,6 +116,7 @@ const MultipleChoiceSubmission: React.FC<QuizItemSubmissionComponentProps> = ({
               <div
                 className={css`
                   padding: 1rem 0;
+                  width: 50ch;
```

Comment:
Does this work on mobile? Maybe something like max-width would scale better.

It's hard to tell just from the pull request since this change doesn't show up in any of the screenshots. Would it be possible to update one of the screenshot tests to include a multiple choice with this much text?